torso-bundle.js

```diff
@@ -305,11 +305,56 @@
           },
 
           /**
+           * Will force the cache to fetch a subset of this collection's tracked ids
+           * @method requesterMixin.fetchSubsetOfTrackedIds
```

Comment:
Then this is fine, I imagine it was someone not running a bundle before merging

+2